components/layout/display_list/builder.rs

```diff
@@ -933,10 +933,14 @@ impl FragmentDisplayListBuilding for Fragment {
             style.get_cursor(CursorKind::Default),
             display_list_section,
         );
-        state.add_display_item(DisplayItem::SolidColor(Box::new(SolidColorDisplayItem {
-            base: base,
-            color: background_color.to_layout(),
-        })));
+
+        // Skip transparent backgrounds.
```

Comment:
We need to include transparent backgrounds to properly support hit testing.

Reply:
Did not know that.